assets/js/components/data-table.js

```diff
@@ -91,13 +92,17 @@ export const getDataTableFromData = ( data, headers, options ) => {
 	} );
 
 	return (
-		<div className={ `googlesitekit-table${ ( options && options.disableListMode ? '' : ' googlesitekit-table--with-list' ) }` }>
-			<table
-				className={ `
-					googlesitekit-table__wrapper
-					googlesitekit-table__wrapper--${ data && data[ 0 ] ? data[ 0 ].length : 1 }-col
-				` }
-			>
+		<div className={ classnames(
+			'googlesitekit-table',
+			{ 'googlesitekit-table--with-list': ! options || ! options.disableListMode }
+		) }>
+			<table className={ classnames(
+				'googlesitekit-table__wrapper',
+				{
+					'googlesitekit-table__wrapper-1': ! data && ! data[ 0 ].length,
```

Comment:
This logic doesn't follow the old one and can't even be true. If `data` is falsey it can't also have an array accessor method. Even an empty array in JS is truthy, so `! data && data[ 0 ]` will always be false… and if `data` is `null` or `undefined` this will throw a `TypeError`. The aim here is to show a single column when the `data` used to populate the table is false-y (`null`/`undefined`) or has no elements inside it (`! data[ 0 ].length`). It's also missing the check to make sure `data[ 0 ]` exists.

This conditional should be: `! data || ! data[ 0 ] || ! data[ 0 ].length`.

Reply:
yes you're right. I copied and pasted.